sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_operations_mixin.py

```diff
@@ -52,16 +52,15 @@ def entities_linking(
         api_version = self._get_api_version('entities_linking')
         if api_version == 'v3.0':
             from .v3_0.operations import TextAnalyticsClientOperationsMixin as OperationClass
-        elif api_version == 'v3.1-preview.1':
```

Comment:
in the old regeneration, I deleted the v3.1-preview.1 generated folder, but didn't regenerate without that API version. This was also a bug before.

Reply:
In other words, we don't support the v3.1-preview.1 API version anymore, correct? Can you remind me if we defaulted to  the preview API version in the preview library, or if the customer still had to opt in to it?

Reply:
Right now we default to api version `v3.1-preview.2`. In the enum, this is jsut exposed to users as `V3_1_PREVIEW`, since @kristapratico and I decided we didn't want to expose both v3.1-preview.1 and 2 for package size reasons / didn't think it was necessary

Reply:
As long as were are defaulting to the latest, I'm good. 

If we *were* defaulting to the latest GA version (which were are not, so this is hypothetical :)) of the service (which has been suggested/discussed in the past), then we would have had to at least tell users to update their code (since they would have had an explicit 3.1 preview parameter when creating clients) - or, possibly, put a `warn` statement about 3.1 no longer being supported and possibly for us to (for the preview version) redirect to the new API version.

Reply:
glad we're defaulting to latest overall then